nipype/interfaces/cat12/preprocess.py

```diff
@@ -0,0 +1,363 @@
+                'locally in T1 or increased in T2/PD images. These so-called WM hyperintensies (WMHs) can lead to ' \
+                'preprocessing errors. Large GM areas next to the ventricle can cause normalization problems. ' \
+                'Therefore, a temporary correction for normalization is useful if WMHs are expected. CAT allows ' \
+                'different ways to handle WMHs: ' \
+                '\n0) No Correction (handled as GM). \n1) Temporary (internal) correction as WM for spatial ' \
+                'normalization and estimation of cortical thickness. \n2) Permanent correction to WM. '
+    wm_hyper_intensity_correction = traits.trait_types.Int(1, field="extopts.WMHC", desc=help_wmhc, usedefault=True)
+
+    help_vox = 'The (isotropic) voxel sizes of any spatially normalised written images. A non-finite value will be ' \
+               'replaced by the average voxel size of the tissue probability maps used by the segmentation.'
+    voxel_size = traits.trait_types.Float(1.5, field="extopts.vox", desc=help_vox, usedefault=True)
+
+    help_resampling = 'Internal resampling for preprocessing.\n The default fixed image resolution offers a good ' \
+                      'trade-off between optimal quality and preprocessing time and memory demands. Standard ' \
+                      'structural data with a voxel resolution around 1 mm or even data with high in-plane resolution' \
+                      ' and large slice thickness (e.g. 0.5x0.5x1.5 mm) will benefit from this setting. If you have' \
+                      ' higher native resolutions the highres option "Fixed 0.8 mm" will sometimes offer slightly' \
+                      ' better preprocessing quality with an increase of preprocessing time and memory demands. In' \
+                      ' case of even higher resolutions and high signal-to-noise ratio (e.g. for 7 T data) the ' \
+                      '"Best native" option will process the data on the highest native resolution. I.e. a resolution' \
+                      ' of 0.4x0.7x1.0 mm will be interpolated to 0.4x0.4x0.4 mm. A tolerance range of 0.1 mm is used' \
+                      ' to avoid interpolation artifacts, i.e. a resolution of 0.95x1.01x1.08 mm will not be ' \
+                      'interpolated in case of the "Fixed 1.0 mm"! This "optimal" option prefers an isotropic voxel ' \
+                      'size with at least 1.1 mm that is controlled by the median voxel size and a volume term that ' \
+                      'penalizes highly anisotropic voxels.' \
+                      'Values:\nOptimal: [1.0 0.1]\nFixed 1.0 mm: [1.0 0.1];\nFixed 0.8 mm:[0.8 0.1]' \
+                      '\nBest native: [0.5 0.1]'
+    internal_resampling_process = traits.trait_types.Tuple(traits.trait_types.Float(1), traits.trait_types.Float(0.1),
+                                                           minlen=2, maxlen=2,
```

Comment:
I'm pretty sure this is redundant, but you may want to check before applying this.

```suggestion
```